lib/broadside/target.rb

```diff
@@ -74,14 +85,29 @@ def family
       "#{Broadside.config.application}_#{@name}"
     end
 
-    def to_h
+    def to_hash
```

Comment:
in https://docs.aws.amazon.com/sdkforruby/api/Aws/ECS/Client.html all of the examples reference `to_h`